pkg/image/apiserver/apiserver.go

```diff
@@ -144,10 +149,29 @@ func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
 		tlsConfig.RootCAs = x509.NewCertPool()
 	}
 
-	if len(c.ExtraConfig.AdditionalTrustedCA) != 0 {
-		if ok := tlsConfig.RootCAs.AppendCertsFromPEM(c.ExtraConfig.AdditionalTrustedCA); !ok {
-			return nil, fmt.Errorf("No valid certificates read from %v", c.ExtraConfig.AdditionalTrustedCA)
+	err = filepath.Walk("/var/run/configmaps/image-import-ca", func(path string, info os.FileInfo, err error) error {
```

Comment:
Can this error if the path does not exist? Noticed that the mount at this path is optional.

Reply:
i'm not sure, but the dir will always exist even if the configmap does not, it'll just be empty (i just checked this).

and worst case an error from filepath.Walk is not being treated as fatal, we just log it.